Chat-Ops-Collector/src/main/java/com/capitalone/dashboard/model/ChatOpsRepo.java

```diff
@@ -0,0 +1,56 @@
+package com.capitalone.dashboard.model;
+
+import org.joda.time.DateTime;
+
+/**
+ * CollectorItem extension to store the chatops serverUrl, roomName and authToken.
+ */
+public class ChatOpsRepo extends CollectorItem {
+    private static final String CHATOPS_SERVER_URL = "chatOpsServerUrl";
```

Comment:
This is a dummy field, once api is updated to register a widget of this type it will not be needed anymore